app/routes/task_routes.py

```diff
@@ -1 +1,98 @@
-from flask import Blueprint
\ No newline at end of file
+from flask import Blueprint, request, Response, abort, make_response
+import requests
+import os
+from .route_utilities import validate_model, create_model
+from datetime import datetime
+from ..models.task import Task
+from ..db import db
+
+bp=Blueprint("tasks_bp", __name__, url_prefix="/tasks")
+
+@bp.post("")
+def create_task():
+    request_body = request.get_json()
+    response = create_model(Task, request_body)
+    return {"task": response[0]}, response[1]
+
+@bp.get("")
+def get_tasks():
+    query = db.select(Task)
+
+    sort_param = request.args.get("sort")
+
+    if sort_param == "asc":
+        tasks = db.session.scalars(query.order_by(Task.title.asc()))
+    elif sort_param == "desc":
+        tasks = db.session.scalars(query.order_by(Task.title.desc()))
+    else:
+        tasks = db.session.scalars(query.order_by(Task.id))
+
+    tasks_response = []
+    for task in tasks:
+        valid_task = task.to_dict()
+        if not has_special_chars(valid_task["title"]):
+            tasks_response.append(valid_task)
+    return tasks_response
+
+def has_special_chars(title):
+    special_chars = "#$%()*+,-./:;<=>?@[\]^_`{|}~"
+    for char in title:   
+        if char in special_chars:
+            raise ValueError
+    return False
+
+@bp.get("/<id>")
+def get_one_task(id):
+    task = validate_model(Task, id)
+    return {"task": task.to_dict()},200 
+
+@bp.put("/<id>")
+def update_task(id):
+    task = validate_model(Task, id)
+    request_body = request.get_json()
+
+    task.title=request_body["title"]
+    task.description=request_body["description"]
```

Comment:
These two lines will throw unhandled exceptions if the request body does not have keys exactly as "title" or "description". How could you update the logic in this route so that, instead of throwing a server error, you could return a nice error response with an appropriate status code?